.papr.yml

```diff
@@ -67,16 +67,15 @@ timeout: 30m
 host:
   distro: fedora/25/atomic
 
-# mount yum.repos.d to get any injected repos from the host, which
+# copy yum.repos.d to get any injected repos from the host, which
 # will point to a closer mirror
 tests:
-  - >
-      docker run --privileged --rm
-      -e RPMOSTREE_COMPOSE_TEST_USE_HOST_REPO=1
-      -v /etc/yum.repos.d:/etc/yum.repos.d:ro
-      -v $(pwd):/srv/code
-      registry.fedoraproject.org/fedora:25 /bin/sh -c
-      "cd /srv/code && ./ci/build.sh && make install && ./tests/compose"
+  - docker run --privileged --rm -d --name composer
+    -e RPMOSTREE_COMPOSE_TEST_USE_HOST_REPO=1 -v $(pwd):/srv/code
+    registry.fedoraproject.org/fedora:25 sleep infinity
```

Comment:
The fahc injection we do is to get rpm-ostree building though, right? So, we'd have to change the dnf configuration as well. Not impossible, but kinda yucky as well.